.github/workflows/ci_levm.yaml

```diff
@@ -58,6 +58,78 @@ jobs:
               echo "Percentage is not 100%."
               exit 1
           fi
+
+  hive-report-creation:
+    uses: ./.github/workflows/common_hive_reports.yaml
+    with:
+      evm: levm
+      job_type: trigger
```

Comment:
nit: there could be a race condition here when two PRs overwrite the same file? It would be safer to append the commit hash in that case